examples/experimental/test/end_to_end/http_server/fetch_ic/test/tests.ts

```diff
@@ -16,7 +16,9 @@ export function getTests(canisterName: string): Test {
     return () => {
         please('prepare browser and pages', async () => {
             browser = await puppeteer.launch({
-                headless: process.env.GITHUB_RUN_ID === undefined ? false : true
+                headless:
+                    process.env.GITHUB_RUN_ID === undefined ? false : true,
```

Comment:
Try removing the `--disable-setuid-sandbox`

Reply:
Done